controllers/rollingupgrade_controller.go

```diff
@@ -1028,14 +1031,16 @@ func (r *RollingUpgradeReconciler) DrainTerminate(
 	ch chan error) {
 
 	// Drain and wait for draining node.
-	err := r.DrainNode(ruObj, nodeName, KubeCtlCall, ruObj.Spec.Strategy.DrainTimeout)
-	if err != nil && !ruObj.Spec.IgnoreDrainFailures {
-		ch <- err
-		return
+	if nodeName != "" {
```

Comment:
Same as the first comment here, let's add some unit tests for this.

Reply:
Done. Added in latest commit.